src/radical/pilot/pilot.py

```diff
@@ -756,7 +756,7 @@ def _control_cb(self, topic, msg_data):
 
     # --------------------------------------------------------------------------
     #
-    def rpc(self, cmd, *args, **kwargs):
+    def rpc(self, cmd, *args, rpc_addr=None, **kwargs):
```

Comment:
the ordering of inputs is not consistent with `Component`
btw, maybe we can not to have `*args`, that's brings some confusion, and instead to have `args=` explicitly

Reply:
We discussed this out of band.  The current version allows a more native-like call invocation, thus leaving as is.